pkg/controllers/csi/driver/server.go

```diff
@@ -43,7 +43,7 @@ import (
 type Server struct {
 	fs      afero.Afero
 	mounter mount.Interface
-	db      metadata.Access
+	db      metadata.GormAccess
```

Comment:
let's use Access, and rename in new ticket ?

Reply:
Done